<launch>
    <node name="hole_broadcaster" pkg="tfr_utilities" output="screen" type="obstacle_broadcaster">
        <rosparam>
            parent_frame: bin_link 
            point_frame: hole
            service_name: localize_hole
            hz: 5 
            diameter: 0.5
        </rosparam>
    </node>
</launch>
